app/src/main/res/layout/item_metadata_add_image.xml

```diff
@@ -1,26 +1,40 @@
 <?xml version="1.0" encoding="utf-8"?>
-<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:tools="http://schemas.android.com/tools"
-    android:layout_width="72dp"
-    android:layout_height="104dp"
-    android:background="@drawable/item_metadata_add_image_background"
-    android:gravity="center"
-    android:orientation="vertical">
 
-    <ImageView
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:paddingStart="1dp"
-        android:src="@drawable/ic_add_photo_rounded"
-        tools:ignore="RtlSymmetry"
-        android:importantForAccessibility="no" />
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
I am not sure why did you change this layout. Also, there's no point in using a `ConstraintLayout` if you only have one child.

Reply:
I did it to align the items inside the RecyclerView in the metadata activity. When I added the dot the add sample item was sticked to the top and I could find how to move it to the bottom